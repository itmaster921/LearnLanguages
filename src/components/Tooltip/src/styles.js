// @flow
import { StyleSheet } from 'react-native';

const styles: StyleSheet.Styles = StyleSheet.create({
  container: {
    ...StyleSheet.absoluteFillObject,
    opacity: 0,
    backgroundColor: 'transparent',
  },
  containerVisible: {
    opacity: 1,
  },
  background: {
    ...StyleSheet.absoluteFillObject,
  },
  tooltip: {
    backgroundColor: 'transparent',
    position: 'absolute',
    shadowColor: 'black',
    shadowOffset: { width: 0, height: 2 },
    shadowRadius: 2,
    shadowOpacity: 0.8,
  },
  content: {
    borderRadius: 4,
    padding: 8,
    backgroundColor: '#fff',
  },
  arrow: {
    position: 'absolute',
    borderTopColor: 'transparent',
    borderRightColor: 'transparent',
    borderBottomColor: 'transparent',
    borderLeftColor: 'transparent',
  },
});

export default styles;
