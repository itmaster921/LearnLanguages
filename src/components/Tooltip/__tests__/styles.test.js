import styles from '../src/styles';

describe('testing styling', () => {
  it('shows stylesheet is as expected', () => {
    expect(styles).toMatchSnapshot();
  });
});
