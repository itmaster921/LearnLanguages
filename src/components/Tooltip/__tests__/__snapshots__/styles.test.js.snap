// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing styling shows stylesheet is as expected 1`] = `
Object {
  "arrow": Object {
    "borderBottomColor": "transparent",
    "borderLeftColor": "transparent",
    "borderRightColor": "transparent",
    "borderTopColor": "transparent",
    "position": "absolute",
  },
  "background": Object {
    "bottom": 0,
    "left": 0,
    "position": "absolute",
    "right": 0,
    "top": 0,
  },
  "container": Object {
    "backgroundColor": "transparent",
    "bottom": 0,
    "left": 0,
    "opacity": 0,
    "position": "absolute",
    "right": 0,
    "top": 0,
  },
  "containerVisible": Object {
    "opacity": 1,
  },
  "content": Object {
    "backgroundColor": "#fff",
    "borderRadius": 4,
    "padding": 8,
  },
  "tooltip": Object {
    "backgroundColor": "transparent",
    "position": "absolute",
    "shadowColor": "black",
    "shadowOffset": Object {
      "height": 2,
      "width": 0,
    },
    "shadowOpacity": 0.8,
    "shadowRadius": 2,
  },
}
`;
